core/src/main/java/uk/gov/justice/services/core/dispatcher/DispatcherFactory.java

```diff
@@ -4,13 +4,23 @@
 
 import uk.gov.justice.services.core.handler.registry.HandlerRegistry;
 
+import javax.inject.Inject;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
 import org.slf4j.Logger;
 
 public class DispatcherFactory {
 
     private static final Logger LOGGER = getLogger(HandlerRegistry.class);
 
+    private ObjectMapper objectMapper;
+
+    @Inject
+    public DispatcherFactory(final ObjectMapper objectMapper) {
```

Comment:
Our default approach to injection is to use package scoped fields, not construction injection

Reply:
As agreed, keep these for now as have to refactor lots of other tests